patches/minecraft/net/minecraft/world/entity/Mob.java.patch

```diff
@@ -24,6 +24,14 @@
           for(ItemEntity itementity : this.f_19853_.m_45976_(ItemEntity.class, this.m_20191_().m_82377_((double)vec3i.m_123341_(), (double)vec3i.m_123342_(), (double)vec3i.m_123343_()))) {
              if (!itementity.m_213877_() && !itementity.m_32055_().m_41619_() && !itementity.m_32063_() && this.m_7243_(itementity.m_32055_())) {
                 this.m_7581_(itementity);
+@@ -517,6 +_,7 @@
+    }
+ 
+    protected void m_7581_(ItemEntity p_21471_) {
++      if(!p_21471_.isPickupAllowed(this, net.minecraftforge.items.pickup.ItemPickupReasons.LIVING_ENTITY)) return; // Forge: Check to see if item currently disallows being picked up for this entity
```

Comment:
Formatting: space between `if` and condition. Additionally, change the comment to:
```diff
-Check to see if item currently disallows being picked up for this entity
+Check to see if the item currently disallows being picked up by this entity
```